packages/plugin-css/src/utils/is-token-match.test.ts

```diff
@@ -1,5 +1,5 @@
 import {describe, expect, test} from 'vitest';
-import {isTokenMatch} from '../src/util.js';
```

Comment:
As a minor note, could you return the tests to the `/test/` folder? It makes it a little easier to exclude those from publishing to npm the way the package is set up. It also keeps all the tests together.

Reply:
aaaand I just realized I’m accidentally publishing plugin tests to npm! Will fix that in the next release. In that case folder doesn’t matter.

Reply:
I think if you're trying to only ship what's necessary you could just specify `package.json` and the `dist` folder. Then anything that doesn't get built by `tsc` should remain out of the production bundle. The `exclude` line I added in `tsconfig.json` prevents these tests from ending up in `dist`.

That being said, if you still prefer they go into the `/test/` folder I'm can move them back no problem.

Reply:
No they’re fine where they are! I realized I’m publishing more to npm than I want to be, so I have cleanup that’s totally unrelated to your PR.